Life-Expectancy-and-GDP-Starter/life_expectancy_gdp.ipynb

```diff
@@ -124,12 +135,25 @@
     "print('DESCRIPTION ABOUT THE DATA:')\n",
     "print('===========================')\n",
     "print(gdp_data.describe(include='all'), \"\\n\")\n",
-    "#reduce life expectancy variable name\n",
     "\n",
+    "# Analyzing unique values in Country and Year variables\n",
+    "print('INFORMATION ABOUT THE COUNTRY AND YEARS:')\n",
+    "print('=================================================================')\n",
+    "print(f'Unique Value of Country: {gdp_data.Country.unique()}')\n",
+    "print(f'Unique Value of Year: {gdp_data.Year.unique()}', \"\\n\")\n",
+    "\n",
+    "# Reduce the large name of United States of America.\n",
+    "gdp_data.loc[gdp_data['Country'] == 'United States of America', 'Country'] = 'U.S.A'\n",
```

Comment:
Change "U.S.A" to "USA". Periods aren't necessary.

Reply:
This is because we had the same idea; you mentioned it when you reviewed Calvin's graphs, and at the same time, I was incorporating it into the changes. Either way is fine; the important thing is to reduce the number of characters in the country name.